### DISCUSSION TITLE
GitHub Training link not working

### DISCUSSION THREAD

**Discussion by esimms999** [community/community#59349] [New to GitHub] [answered]
I cannot get to the training site; has the URL changed?

![image](https://github.com/community/community/assets/10915780/e1bde3c5-9f7d-48df-8710-26a9db18a177)

**Comment by esimms999** [comment]
I think that all of this training has been moved under https://skills.github.com , e.g. https://github.com/skills/introduction-to-github

I will leave this discussion open, for now, in case someone can provide info on when this changed.

**Comment by airtower-luna** [comment, marked as answer]
Your assumption is correct: https://github.blog/changelog/2022-09-01-deprecating-learning-lab/

**Comment by esimms999** [comment]
@airtower-luna Thanks! That says it all. I will close this discussion.
